keyboards/linworks/em8/keymaps/via/rules.mk

```diff
@@ -0,0 +1 @@
+VIA_ENABLE = yes
```

Comment:
Please fix the missing newline at the end of this file.

Reply:
Thx, was added.